src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,46 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * This method triggers the callback for onItemSelected with the index and "false" parameter indicating that
```

Comment:
`This method` is redundant. Be consistent with how we usually comment methods. 'Triggers the ..`